flink/v1.14/flink/src/main/java/org/apache/iceberg/flink/FlinkConfigOptions.java

```diff
@@ -19,15 +19,11 @@
 
 package org.apache.iceberg.flink;
 
-
 import org.apache.flink.configuration.ConfigOption;
 import org.apache.flink.configuration.ConfigOptions;
 
 public class FlinkConfigOptions {
 
-  private FlinkConfigOptions() {
```

Comment:
why removing this private constructor?

Reply:
It wasn't removed,  the code auto-formatted moved it below.

Reply:
I think we should revert this change. Iceberg coding style has the private constructor at the beginning. see `TableProperties` class.

Reply:
updated